Classes/WPEditorView.m

```diff
@@ -1184,8 +1187,6 @@ - (void)insertLocalImage:(NSString*)url uniqueId:(NSString*)uniqueId
 {
     NSString *trigger = [NSString stringWithFormat:@"ZSSEditor.insertLocalImage(\"%@\", \"%@\");", uniqueId, url];
     [self.webView stringByEvaluatingJavaScriptFromString:trigger];
-    
-    [self workaroundiOS7FocusIssueAfterHidingBehindAnotherVC];
```

Comment:
Since the method being called is not used anywhere else anymore, you should remove it from the sources.